src/main/java/com/amazon/opendistroforelasticsearch/sql/utils/QueryDataMask.java

```diff
@@ -0,0 +1,83 @@
+/*
+ *
+ *  * Licensed under the Apache License, Version 2.0 (the "License").
+ *  * You may not use this file except in compliance with the License.
+ *  * A copy of the License is located at
+ *  *
+ *  *    http://www.apache.org/licenses/LICENSE-2.0
+ *  *
+ *  * or in the "license" file accompanying this file. This file is distributed
+ *  * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *  * express or implied. See the License for the specific language governing
+ *  * permissions and limitations under the License.
+ *
+ */
+
+package com.amazon.opendistroforelasticsearch.sql.utils;
+
+import com.alibaba.druid.sql.ast.expr.SQLQueryExpr;
+import com.amazon.opendistroforelasticsearch.sql.rewriter.identifier.RemoveSensitiveDataRule;
+import java.util.HashSet;
+import java.util.Set;
+
+import static com.amazon.opendistroforelasticsearch.sql.utils.Util.toSqlExpr;
+
+/**
+ * Utility class for mask sensitive info in incoming sql queries
```

Comment:
np:
`for` -> `to`
`info` -> `information`
`sql` -> `SQL`

Same elsewhere.

Reply:
Will do, thanks!